inputfiles/addedTypes.json

```diff
@@ -89,6 +89,120 @@
         ]
     },
     {
+        "kind": "typedef",
+        "flavor": "Worker",
+        "name": "HTMLOrSVGImageElement",
+        "type": "HTMLImageElement | SVGImageElement"
+    },
+    {
+        "kind": "typedef",
+        "flavor": "Worker",
+        "name": "CanvasImageSource",
+        "type": "HTMLOrSVGImageElement | HTMLVideoElement | HTMLCanvasElement | ImageBitmap"
+    },
+    {
+        "kind": "typedef",
+        "flavor": "Worker",
+        "name": "ImageBitmapSource",
+        "type": "CanvasImageSource | Blob | ImageData"
```

Comment:
consolodate all the types in one type def. we try to avoid polluting the global namespace.

```ts
type ImageBitmapSource = HTMLImageElement | SVGImageElement | HTMLVideoElement | HTMLCanvasElement | ImageBitmap; | Blob | ImageData;
```